### PR TITLE
command descriptors article

### PR DESCRIPTION
**Pull request by DKaramfilov**

### CODE REVIEW

File: controls/radspreadsheet/features/command-descriptors.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Command Descriptors
+page_title: Command Descriptors
+description: Describes how you can override the default command descriptors and create custom commands
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/command-descriptors.md:4]: Describes how you can override the default command descriptors of RadSpreadsheet for WPF and create custom commands.

File: controls/radspreadsheet/features/command-descriptors.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+The second step is to set the newly created class to the active worksheet editor. This should be done each time the editor is changed, this is why you are going to use the __ActiveSheetEditorChanged__ event to set the new descriptors. 
+
+#### __C# Change the default command descriptors__
+
+{{region radspreadsheet-command-descriptors_1}}
+
+    private void RadSpreadsheet_ActiveSheetEditorChanged(object sender, EventArgs e)
+    {
+        RadWorksheetEditor editor = (RadWorksheetEditor)this.radSpreadsheet.ActiveSheetEditor;
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/command-descriptors.md:76]: A null check might be useful here, to guarantee that the editor is already initialized and won't be null

File: controls/radspreadsheet/features/command-descriptors.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+
+{{endregion}}
+
+
+
+The second step is to set the newly created class to the active worksheet editor. This should be done each time the editor is changed, this is why you are going to use the __ActiveSheetEditorChanged__ event to set the new descriptors. 
+
+#### __C# Change the default command descriptors__
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/command-descriptors.md:70]: Fix the formatting of the caption

File: controls/radspreadsheet/features/command-descriptors.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+* __Command__(ICommand): The command to be executed.
+* __SelectedValue__(object): Gets or sets the selected value (if available for the current command). 
+
+## Custom Command Descriptors
+
+The following section demonstrates how you can create a custom command along with the respective command descriptor.
+
+First you need to inherit the __WorksheetCommandDescriptors__ class. This will allow you to override the default commands properties and return your custom command. In this case, you will change the open file command with a custom one. If you want to override the key combination that triggers this command you should register the new command as well. 
+
+#### __C# Create a class that inherits WorksheetCommandDescriptors__
```
Comment (TanyaDimitrova) [inline controls/radspreadsheet/features/command-descriptors.md:28]: Fix the formatting of the caption

**Review by TanyaDimitrova — CHANGES_REQUESTED** [review]

File: controls/radspreadsheet/features/command-descriptors.md (hunk 1 above)
Comment (DKaramfilov) [reply controls/radspreadsheet/features/command-descriptors.md]: Fixed

File: controls/radspreadsheet/features/command-descriptors.md (hunk 2 above)
Comment (DKaramfilov) [reply controls/radspreadsheet/features/command-descriptors.md:76]: Fixed

File: controls/radspreadsheet/features/command-descriptors.md (hunk 3 above)
Comment (DKaramfilov) [reply controls/radspreadsheet/features/command-descriptors.md]: Fixed

File: controls/radspreadsheet/features/command-descriptors.md (hunk 4 above)
Comment (DKaramfilov) [reply controls/radspreadsheet/features/command-descriptors.md]: Fixed

**Pull request merged**